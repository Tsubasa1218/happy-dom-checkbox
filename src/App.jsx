function App() {
  return (
    <input
      type="checkbox"
      checked={true}
      onChange={() => {
        /*noop*/
      }}
    />
  );
}

export default App;
