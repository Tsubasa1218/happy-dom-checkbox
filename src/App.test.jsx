import { render } from "@testing-library/react";
import { test, expect } from "vitest";

import App from "./App";

test("Example", () => {
  const { asFragment } = render(<App />);

  expect(asFragment()).toMatchSnapshot();
});
