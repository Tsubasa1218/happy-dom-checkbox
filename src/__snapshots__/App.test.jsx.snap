// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Example 1`] = `
<DocumentFragment>
  <input
    type="checkbox"
  />
</DocumentFragment>
`;
